shared-components/tl-tournament-card.js

```diff
@@ -251,26 +208,51 @@ export class TournamentCard extends LitElement {
                 return html` <h4>loading</h4> `;
             case State.LOADED:
                 return html`
-                    <h4>Title: ${this.tournament.name}</h4>
+                    <img src=${this.tournament.eventImage} />
+                    <h4 class="event-title">${this.tournament.name}</h4>
                     <!-- FIXME: change the value of the substring to match official given value -->
-                    <div class="paragraph-large">Short description: ${this.tournament.description.substring(0, 30)}</div>
-                    <!-- FIXME: show dates like a countdown -->
-                    <div class="paragraph-medium">Event dates: ${this._formatDates()}</div>
-                    <!-- FIXME: show real prize amount -->
-                    <!-- FIXME: use the localization API to generate the plural of the currency symbol -->
-                    <div class="paragraph-large">Prize money: ${this.tournament.fee}&nbsp;${this.tournament.currency.symbol}s</div>
-                    <mwc-button class="details-button button-label-large" label="Details" outlined @click="${(event) => this._handleDetailsButton(event)}"></mwc-button>
+                    <div class="paragraph-large event-text">
+                        ${this.tournament.description.substring(0, 30)}
+                    </div>
+                    <tl-tournament-countdown
```

Comment:
Logically the `<tl-tournament-countdown/>` should be the first child in that list, right? It appears before the image... I know you use the `position: absolute` trick to fix it but it's not necessary...